tests/bud.bats

```diff
@@ -390,6 +390,23 @@ _EOF
   assert "$output" !~ "unwanted stage"
 }
 
+# Note: Please skip this tests in case of podman-remote build
+@test "build: test race in updating image name while performing parallel commits" {
+  # Run 25 parallel builds using the same Containerfile
```

Comment:
Try `_prefetch alpine`?

Can't help you with the rest of the errors, though.

Reply:
Strange part is the test passes on local and fails on CI across all VM, which makes me feel like something in the test is not right and only available on my local. I hope its the missing `_prefetch alpine`. Amended.

Reply:
I think the issue is we have timeout too short for CI
```console
[+0438s] # timeout: sending signal TERM to command ‘/var/tmp/go/src/github.com/containers/buildah/tests/./../bin/buildah’
[+0438s] # timeout: sending signal TERM to command ‘/var/tmp/go/src/github.com/containers/buildah/tests/./../bin/buildah’
[+0438s] # timeout: sending signal TERM to command ‘/var/tmp/go/src/github.com/containers/buildah/tests/./../bin/buildah’
[+0438s] # timeout: sending signal TERM to command ‘/var/tmp/go/src/github.com/containers/buildah/tests/./../bin/buildah’
[+0438s] # timeout: sending signal TERM to command ‘/var/tmp/go/src/github.com/containers/buildah/tests/./../bin/buildah’
[+0438s] # container exited on killed
[+0438s] # container exited on killed
[+0438s] # container exited on killed
```

Reply:
Seems likely.

Please don't forget to address all the other failures.